src/platform/windows/display_device/windows_utils.cpp

```diff
@@ -0,0 +1,603 @@
+// lib includes
+#include <boost/algorithm/string.hpp>
+#include <boost/uuid/name_generator_sha1.hpp>
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+// standard includes
+#include <iostream>
+#include <system_error>
+
+// local includes
+#include "src/logging.h"
+#include "src/utility.h"
+#include "windows_utils.h"
+
+// Windows includes after "windows.h"
+#include <SetupApi.h>
+
+namespace display_device {
+
+  namespace w_utils {
+
+    namespace {
+
+      std::string
+      convert_to_string(const std::wstring &str) {
```

Comment:
You can drop this and switch to `platf::to_utf8()`/`platf::from_utf8()`. We shouldn't use `std::wstring_convert` or `std::codecvt_utf8` anymore.